WebRoot/js/ajax/config/msgs/AjxMsg_vi.properties

```diff
@@ -0,0 +1,157 @@
+# 
```

Comment:
I have the same in zm-admin-ajax\WebRoot\js\ajax\config\msgs\. Only en-US file so removing _vi from there as well.